k8s/plugin/Cargo.toml

```diff
@@ -19,7 +19,7 @@ tls = [ "openapi/tower-client-tls", "rest-plugin/rls" ]
 [dependencies]
 openapi = { path = "../../dependencies/control-plane/openapi", default-features = false, features = [ "tower-trace" ] }
 utils = { path = "../../dependencies/control-plane/utils/utils-lib" }
-rest-plugin = { path = "../../dependencies/control-plane/control-plane/plugin", default-features = false }
+rest-plugin = { path = "../../dependencies/control-plane/control-plane/plugin", default-features = true }
```

Comment:
this should probably remain set to false so it may be controlled by the feature flags.